static/src/javascripts-legacy/projects/common/modules/experiments/tests/paid-content-vs-outbrain.js

```diff
@@ -24,12 +24,20 @@ define([
         variants: [
             {
                 id: 'paid-content',
-                test: function () {
+                // TODO: ask someone about this
```

Comment:
@regiskuckaertz do we still need this file? I noticed the test function does nothing and it's switched off on the switchboard